manifests/varnish-modules.pp

```diff
@@ -0,0 +1,34 @@
+# Class: yumrepos::varnish-modules
```

Comment:
Change this to match class name below. Also rename file to match: `varnish-41-modules.pp`